diff --git a/protocol/meson.build b/protocol/meson.build
index bdd322ac..e50b23bb 100644
--- a/protocol/meson.build
+++ b/protocol/meson.build
@@ -1,4 +1,4 @@
-dep_scanner = dependency('wayland-scanner', native: false)
+dep_scanner = dependency('wayland-scanner', native: true)
 prog_scanner = find_program(dep_scanner.get_pkgconfig_variable('wayland_scanner'))
 
 dep_wp = dependency('wayland-protocols', version: '>= 1.24')
